.github/workflows/test.yaml

```diff
@@ -202,33 +202,57 @@ jobs:
           fail_action: false
 
   e2e-tests:
+    name: 🧪 e2e tests ${{ matrix.project }}
     needs:
-      ["backend-docker-build", "frontend-docker-build", "install-dev-tools"]
-    timeout-minutes: 60
+      - backend-docker-build
+      - frontend-docker-build
+      - install-dev-tools
     runs-on: ubuntu-latest
+    timeout-minutes: 60
     strategy:
       fail-fast: false
       matrix:
-        shardIndex: [1, 2, 3, 4]
-        shardTotal: [4]
+        include:
+          - project: chromium
+            os: ubuntu-latest
+            cache_dir: ~/.cache/ms-playwright
+          - project: firefox
+            os: ubuntu-latest
```

Comment:
I wonder if we should test in Edge given that this is a BC Gov project and many people are likely using Edge.

Reply:
Good idea...but, `Edge`  😬

Reply:
Haha I know, I only brought it up because of that recent post that some users were getting errors logging into Edge 🙃